sphinx_js/typedoc.py

```diff
@@ -167,19 +173,27 @@ def make_description(self, comment):
                 comment.get('text', '')
             ])
 
-    def make_param(self, param):
+    def make_param(self, param, parent_comment=None):
         """Construct a jsdoc parameter entry"""
         typeEntry = param.get('type')
+        description = self.make_description(param.get('comment'))
+        name = param.get('name')
+        if parent_comment is not None and parent_comment.get('tags') is not None:
+            for tag in parent_comment.get('tags'):
+                tag_text = tag.get('text')
+                if tag.get('tag') == 'arg' and tag_text.startswith(name):
+                    tag_text.replace(name, '', 1)
+                    description = tag_text
```

Comment:
Could you add a comment to the this if block? IMO it's not immediately obvious why it is needed.

Reply:
Sometimes `typedoc` writes arguments to `tags` of the parent, and this code extracts them.
I have an example at my work PC and will have a look at it + implement a test and comments toworrow.

Reply:
This code extracted the parameter docstring if instead of the keyword `@param`, the aliases `@arg` or `@argument` were used.